app/jobs/clone_ansible_role.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CloneAnsibleRole < ::ApplicationJob
+  queue_as :default
+
+  def humanized_name
+    'Clone Ansible Role from VCS'
```

Comment:
Aren't these typically translatable?
```suggestion
    _('Clone Ansible Role from VCS')
```